phalcon/cache/backend/file.zep

```diff
@@ -160,6 +173,11 @@ class File extends Backend implements BackendInterface
 					let ret = frontend->afterRetrieve(cachedContent);
 					return ret;
 				}
+			} else {
+				/**
+				 * As the content is expired, deleting the key
+				 */
+				this->delete(keyName);
```

Comment:
I don't think we should return something specific here. The function wasn't returning anything anyway, I've just included this delete call to make sure that the cache file is deleted.